src/index.js

```diff
@@ -0,0 +1,98 @@
+// import _ from 'lodash';
+import './style.css';
+import { mousedown, dragndrop } from './drag';
+import { status, prepopstatus } from './status';
+import {
+  removecompleted, removeAll, Duty, edit, removetask,
+} from './todo';
+
+const form = document.getElementById('form');
+const taskinput = document.querySelector('.taskadder');
+const sync = document.querySelector('.sync');
+const entericon = document.querySelector('.enter-icon');
+const deletecompleted = document.getElementById('delcompleted');
+const list = [];
+let displayedList;
+
+const todoList = (arr) => {
+  arr.forEach((element) => {
+    const duties = document.getElementById('duties');
+    // Create task li //
+    duties
+      .appendChild(document.createElement('li'))
+      .setAttribute('id', element.index);
+    const task = document.getElementById(element.index);
```

Comment:
Optional

Optional suggestions will not stop your code from being approved, but are strongly suggested to do it!

- [ ] Reduce the scope of the "add event listener" functions of your code

I notice you use the ID of the elements to assign the functions, that is good, but actually you are searching them in all the document

i suggest to you to reduce the search scope searching only in the task container, that can be done using list_variable.document.getElementbyId instead of document, it will make your code work more faster

Reply:
Resolved as required